--- ProxifyLocals.lua.orig	2025-11-16 19:00:00.000000000 +0000
+++ ProxifyLocals.lua	2025-11-16 19:30:00.000000000 +0000
@@ -717,6 +717,8 @@
     table.insert(ast.body.statements, 1, Ast.LocalVariableDeclaration(self.setMetatableVarScope, {self.setMetatableVarId}, {
         Ast.VariableExpression(self.setMetatableVarScope:resolveGlobal("setmetatable"))
     }));
+    
+    return ast;
 end
 
 return ProifyLocals;
